ci/ci_run_all_tests.sh

```diff
@@ -17,11 +17,14 @@ set -e
 
 cd $TEBASEDIR
 
+# Check if clients are installed
+dpkg -s mosquitto-clients
+
 # Run all PySys tests
 
-python3 -m venv ~/env-pysys
+python3 -mvenv ~/env-pysys
```

Comment:
Accidentally removed that space in `-m venv` or this is needed? Asking because I've been using the command with the space and it has worked for me. Wondering what's the issue here.

Reply:
It is not needed, but I will add it again to avoid confusion. Kind of slipped in while merging code.